omod/src/main/webapp/pages/results.gsp

```diff
@@ -60,7 +75,39 @@
             iDisplayLength: 10,
             sPaginationType: "full_numbers",
             bAutoWidth: false,
-            sDom: 't<"fg-toolbar ui-toolbar ui-corner-bl ui-corner-br ui-helper-clearfix datatables-info-and-pg "ip>'
+            sDom: 'Bt<"fg-toolbar ui-toolbar ui-corner-bl ui-corner-br ui-helper-clearfix datatables-info-and-pg "ip>',
+            buttons: [
+                {
+                    extend: 'excelHtml5',
+                    text: '<i class="icon-file-excel" title="Export as Excel"></i>',
+                    className: 'export-btn',
+                    exportOptions: {
+                        columns:[ 4,1,2,3,5 ]
+                    },
+                    title: 'Data Quality Violations',
```

Comment:
Its the title of list which gets exported. You can see this title in the image I've attached.